packages/node_modules/@webex/webex-core/src/lib/services/service-catalog.js

```diff
@@ -106,6 +108,135 @@ const ServiceCatalog = AmpState.extend({
     return this;
   },
 
+  /**
+   * Search over all service groups to find a cluster id based
+   * on a given url.
+   * @param {string} url - Must be parsable by `Url`
+   * @returns {string} - ClusterId of a given url
+   */
+  findClusterId(url) {
+    const incomingUrlObj = Url.parse(url);
+    let clusterId;
+    let serviceUrlObj;
+
+    for (const key of Object.keys(this.serviceGroups)) {
+      for (const service of this.serviceGroups[key]) {
```

Comment:
Get rid of the surrounding for loop and change this to `for (const service of Object.values(this.serviceGroups)) {`